portal/eproms/views.py

```diff
@@ -352,14 +352,17 @@ def resources():
     org = user.first_top_organization()
     if not org:
         abort(400, 'user must belong to an organization')
-    resources_data = get_any_tag_data(
-        '{} work instruction'.format(org.name.lower()))
+    asset_title = '{} work instruction'.format(org.name.lower())
```

Comment:
should this string be translated?
`'{} work instruction'`

Reply:
I believe Diana mentioned that no needs for it to be translated:  https://mo-programs.slack.com/archives/C6JFH0C4A/p1558306641000700